features/checkout/seeing_order_summary/seeing_order_locale.feature

```diff
@@ -19,9 +19,17 @@ Feature: Seeing order locale on order summary page
         Then I should be on the checkout summary step
         And my order's locale should be "English (United States)"
 
-    @ui
+    @ui @api
     Scenario: Seeing order locale on the order summary page after change channel locale
         Given I have product "Stark T-Shirt" in the cart
         When I proceed through checkout process in the "French (France)" locale
         Then I should be on the checkout summary step
-        And my order's locale should be "français (France)"
+        And my order's locale should be "French (France)"
+
+    @api
+    Scenario: Seeing order locale on the order summary page after choosing channel locale
+        Given I have cart picked up in the "French (France)" locale
+        And I have product "Stark T-Shirt" in the cart
+        When I proceed through checkout process
```

Comment:
It should rather check if we can pickup-cart in locale through API. Also, in the long term, we should have a validation check, if the locale is available during update